pom.xml

```diff
@@ -19,11 +19,12 @@
 
   <modelVersion>4.0.0</modelVersion>
 
-  <parent>
-    <groupId>org.apache</groupId>
-    <artifactId>apache</artifactId>
-    <version>23</version>
-  </parent>
+    <parent>
+        <groupId>org.apache</groupId>
+        <artifactId>apache</artifactId>
+        <version>23</version>
+        <relativePath/>
```

Comment:
This pom has been published into the maven central repo and no need to add relative path. If you can not download it, pls consider check your network connection or configure a maven mirror.